src/results/electric_tariff.jl

```diff
@@ -14,6 +14,22 @@
 - `year_one_export_benefit_before_tax` export credits over the first year, before considering tax benefits. A positive value indicates a benefit. 
 - `lifecycle_coincident_peak_cost_after_tax` lifecycle coincident peak charge in present value, after tax
 - `year_one_coincident_peak_cost_before_tax` coincident peak charge over the first year
+
```

Comment:
Are all of these "_before_tax" or "_after_tax"? I realize all of the charges/rates will be before tax, so this is mainly a question for the costs ($).

Reply:
All costs are before taxes, just raw cost of electricity. Updated the code to reflect this.